app/bundles/InstallBundle/Controller/InstallController.php

```diff
@@ -21,7 +21,7 @@
 use Symfony\Component\HttpFoundation\JsonResponse;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\HttpKernel\Event\FilterControllerEvent;
-use Symfony\Component\Security\Core\Encoder\PasswordEncoderInterface;
+use Symfony\Component\Security\Core\Encoder\UserPasswordEncoder;
```

Comment:
This use statement is not used anywhere and can be removed. This is just a recommendation, nothing that should block the PR merge.